code/_onclick/item_attack.dm

```diff
@@ -68,15 +72,22 @@
 	add_logs(user, M, "attacked", src.name, "(INTENT: [uppertext(user.a_intent)]) (DAMTYPE: [uppertext(damtype)])")
 	add_fingerprint(user)
 
+	user.adjustStaminaLoss(w_class*1.25)//CIT CHANGE - makes attacking things cause stamina loss

```

Comment:
items should have a 1-100 weight imo, instead of just w_class, w_class is used for balancing storage as much as weight, because i'd imagine a lightsabre, although "bulky", isn't actually that heavy.

Reply:
long term stretch goal i guess i don't expect you to change everything's weights lol
or just make weights auto assigned if there's no specified weight and have it default to null =p

Reply:
:+1: